pages/Mining.qml

```diff
@@ -251,7 +251,14 @@ Rectangle {
 
     function updateStatusText() {
         if (appWindow.isMining) {
-            statusText.text = qsTr("Mining at %1 H/s").arg(walletManager.miningHashRate()) + translationManager.emptyString;
+            var blockTime = 120;
+            var blocksPerDay = 86400 / blockTime;
+            var globalHashRate = walletManager.networkDifficulty() / blockTime;
+            var userHashRate = walletManager.miningHashRate();
```

Comment:
Doesn't handle zero hashrate case.

Reply:
I added a "mining temporarily suspended" message for the zero hashrate case.

Reply:
Why temporarily suspended? Why not something like "You are not currently mining"

Reply:
This message shows when background mining is enabled, but temporarily suspended because some other program is using a lot of CPU. There is a different message when mining is not enabled ("Not mining", if I recall correctly).